### ISSUE TITLE
Slash in linear fraction not positioned properly

### ISSUE THREAD

**Issue by Alexical** [khaledhosny/xits#55]
Using Word 2016 equation editor, the slash in "linear fraction" does not appear to be positioned properly.
![image](https://user-images.githubusercontent.com/22560427/36711117-ac886abc-1bbc-11e8-8446-b25a4c0f3d65.png)
It looks fine with the default math font (Cambria Math). Not sure if it is an issue with Word or with XITS font.

**Comment by khaledhosny** [comment]
I need to know which slash character is used here.

**Comment by Alexical** [comment]
I don't know how to find that out as the character can't be selected in Word. If it helps, this is the exact fraction I'm referring to:
![image](https://user-images.githubusercontent.com/22560427/36718279-2a498778-1bdc-11e8-8926-2aac87ee7779.png)

**Comment by khaledhosny** [comment]
Might be fraction slash, and then that would be by design as fraction slash shouldn’t be used with normal-sized digits.

**Comment by Alexical** [comment]
Hm, that makes sense. Smaller digits would be fine but I can't work out how to achieve that in Word at the moment, without shrinking the entire fraction (defeating the purpose).

For reference, here is a comparison between XITS and Cambria Math:
![image](https://user-images.githubusercontent.com/22560427/36721189-2af83de6-1be5-11e8-8725-32407a25ef06.png)

**Comment by Alexical** [comment]
I guess my question is, whether you would consider supporting these kinds of fractions in Word out-of-the-box, or suggest the user to find a workaround?

**Comment by khaledhosny** [comment]
I need first to know the character for sure. Can you attach a document with this fraction, I might be able to extract the character from the XML source.

**Comment by Alexical** [comment]
Absolutely: [XITS Linear Fractions.docx](https://github.com/khaledhosny/xits/files/1762077/XITS.Linear.Fractions.docx)

**Comment by Alexical** [comment]
After a bit of poking around, I think I have been able to identify the slash character.

First I converted the document to PDF using Microsoft Print to PDF: [XITS Linear Fractions.pdf](https://github.com/khaledhosny/xits/files/1773185/XITS.Linear.Fractions.pdf) I was then able to select the slash character in Adobe Reader and copy it to clipboard. According to Free Clipboard Viewer, the character codes are:

ANSI: `2E 00`
OEM: `2F 00`
Unicode: `2044 0000`

Is that useful?

**Comment by khaledhosny** [comment]
Thanks, this is the fraction slash like I guessed and it has negative side bearing intentionally to combine with superscript and subscript glyphs, but I guess this is not needed in math font since the layout engine will do the kerning for us. Please check if it works now.

**Comment by khaledhosny** [comment]
Please also check that non-linear fractions are not broken by this.
